packages/round-manager/src/features/api/__tests__/utils.test.ts

```diff
@@ -6,6 +6,9 @@ enableFetchMocks()
 
 const fetchMock = fetch as FetchMock
 
+const GOERLI_CHAIN_ID = 5;
+const OPTIMISM_KOVAN_CHAIN_ID = 69;
+const OPTIMISM_MAINNET_CHAIN_ID = 10;
```

Comment:
Is it worth making this into a function in utils where you pass the chain name and get the id ?
My hunch is that this will list grow as we expand and we might have to duplicate these constants in other files

Reply:
I actually created a ChainId enum instead since we aren't currently converting chain name to id directly anywhere in the code.